rpc/core/tx.go

```diff
@@ -214,6 +221,40 @@ func ProveShares(
 	return &ctypes.ResultShareProof{Proof: shareProof}, nil
 }
 
+// TxStatus retrieves the status of a transaction given its hash. It returns a ResultTxStatus
+// containing the height and index of the transaction within the block(if committed)
+// or whether the transaction is pending, evicted from the mempool, or otherwise unknown.
+func TxStatus(ctx *rpctypes.Context, hash []byte) (*ctypes.ResultTxStatus, error) {
```

Comment:
As a follow up issue: we will probbaly need to allow for the user to request a proof of the tx's inclusion if it has been committed

Reply:
https://github.com/celestiaorg/celestia-core/issues/1389